Sources/GRInputField/SwiftUI/InputField.swift

```diff
@@ -0,0 +1,279 @@
+//
+//  InputField.swift
+//  benu
+//
+//  Created by Filip Šašala on 10/04/2024.
+//
+
+import Combine
+import GoodExtensions
+import SwiftUI
+
+// MARK: - View
+
+public struct InputField: UIViewRepresentable {
+
+    // MARK: - Wrappers
+
+    @Binding public var text: String
+    @Binding private var validityGroup: ValidityGroup
+
+    // MARK: - Variables
+
+    public let title: String?
+    public let placeholder: String?
+    public let hint: String?
+    public let rightButton: Supplier<UIButton>?
+
+    private var traits: InputFieldTraits
+    @ValidatorBuilder private var criteria: Supplier<Validator>
+    private var returnAction: VoidClosure?
+    private var resignAction: VoidClosure?
+
+    // MARK: - Initialization
+
+    public init(
+        text: Binding<String>,
+        title: String? = nil,
+        placeholder: String? = nil,
+        hint: String? = " ",
+        rightButton: Supplier<UIButton>? = nil
```

Comment:
Nebolo by lepsie keby tam mohlo ist akekolvek viewcko?

Reply:
jasne ze ano, planujem to prerobit na nejaku `@ViewBuilder` closuru aby si tam vedel dat cokolvek 😄

Reply:
bude v Swift 6 verzii